src/sphinx_codeautolink/extension/block.py

```diff
@@ -159,10 +178,11 @@ def parse_source(
             return
 
         source = node.children[0].astext()
-        transformer = (
-            self.custom_blocks.get(language, default_transform) or default_transform
-        )
-        source, clean_source = transformer(source)
+        transformer = self.custom_blocks.get(language)
```

Comment:
```suggestion
        transformer = self.custom_blocks[language]
```
If we get without default, it might as well be a subscript.

Reply:
Your argumentation is wrong, but you are right.
Since all languages are now handled consistently in a single list, we know that all of them are present in the dictionary.

Reply:
Wait, what do you mean?